project/CONTRIBUTING.md

```diff
@@ -97,6 +97,21 @@ Some rules have to be respected about the test:
 * Most of the time, the test class should have the same name as the targeted class, suffixed by `Test`.
 * The `@expectedException*` annotations are prohibited. Use `PHPUnit_Framework_TestCase::setExpectedException()`.
 
+Setup phpunit to run the tests using:
+```
+composer install
+wget "https://phar.phpunit.de/phpunit.phar" --output-document="/usr/local/bin/phpunit"
```

Comment:
This should not be here, agree with @core23.

If you really want to specify it, it should be a indication of global phar usage with a link to official instructions.